### PR TITLE
Elaborate replacement of PLA gantry parts

### LINKED ISSUE #106: Add gantry disassembly documentation for Vz235
As mentioned in issue #91, there is no documentation for Vz235 on how to replace the PLA printed parts on the Mellow Vz235 kit.

https://docs.vzbot.org/vz235_mellow/gantry vs
https://docs.vzbot.org/vz330_mellow/gantry/change_parts

We would need matching CAD renders in order to update the page

### PR DESCRIPTION
**Pull request by tehmaze**
Elaborated on replacing the PLA parts as per issue #91

### CODE REVIEW

File: docs/vz330_mellow/gantry/disassemble.md
Comment (HumbleDeer) [inline docs/vz330_mellow/gantry/disassemble.md]: typo here :)
te -> the

**Review by HumbleDeer — CHANGES_REQUESTED** [review]
All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them. 

Willing to do this myself too, no big deal.

File: docs/vz330_mellow/gantry/disassemble.md
Comment (HumbleDeer) [reply docs/vz330_mellow/gantry/disassemble.md]: Oh -- Having the artificial line length limits isn't really required for our use cases. Any particular reason for using them? 't makes the diff look funny :^)

**Comment by tehmaze** [comment]
> All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them.

Can we make that a snippet that can be recycled in other places?

**Comment by HumbleDeer** [comment]
> > All that remains would be to copy these changes to the other manuals that have a disassembly/replacement step in them.
> 
> Can we make that a snippet that can be recycled in other places?

yesn't.

Using our system it's not worth doing that. We're better off keeping track of where we have these things posted by having the same stuff posted on the same pages with the same titles.

**Review by HumbleDeer — APPROVED** [review]
It's not ideal, but we can revise this later on or in the future. It's a start! I'll let you decide whether you want to merge this in or wait and polish it up a little more.

**Comment by HumbleDeer** [comment]
Please see https://github.com/tehmaze-labs/vzbot-docs/pull/2 before merging this PR

**Pull request merged**
